enhancements/platforms/ovirt-rhv-ipi.md

```diff
@@ -0,0 +1,277 @@
+and then choose `ovirt` the installation will ask for all the relevant details\
+of the installation: **url**, **user** and **password** of ovirt api. The installer \
+will validate it can communicate with the api, otherwise it will fail to proceed.\
+Next the installer will ask to choose the ovirt **cluster**, where the VMs will be\
+created, and next would be a *template* from that cluster for those VMs.\
+*Note* that it is the user's responsibility to upload the relevant RHCOS template\
+to ovirt. This may change in the future.
+[What is a VM Template][template-what] and [how to upload a template from qcow image?][template-upload]\
+With that the survey continues to the general cluster name, domain name, and\
+the rest of the non-ovirt specific question.
+
+2. Resource creation - terraform
+
+All the information from the survey is serialized by the installer into the file\
+`terraform.ovirt.auto.tfvars.json` under the current directory and then it invokes\
+terraform, which automatically load `.auto.tfvars.json`. `terraform.tfstate` is\
+the actual declaration of the resources and with it terraform creates the VMs.\
+
+3. Bootstrap
+
+The bootstrap VM has a huge Ignition config set using terraform and is visible\
+in the `terraform.tfstate` file. oVirt boots that VM with that content and the\
+bootstraping begins when the `bootkube.service` systemd service starts.\
+The bootstrap job is to:
+    1. run a DNS service which will listen on the DNS VIP supplied by the survey\
+    to allow internal resolution of the api and resolution of each node's FQDN\
+    2. run `keepalived` to activate the DNS VIP and API VIP on the machine nic as\
+    secondary IP. 
+    3. run and expose machine configs, using a static pod of Machine Config Server\
+    listening on the API VIP
+    4. load and bootstrap `etcd` service and help sign `etcd` members on the masters\
+    and boot statically the rest of the cluster pods.
+    5. pivot to a real master after its done
+
+4. Masters bootstrap
+
+Master VMs are booting using a stub Ignition config that are waiting early in\
+the Ignition service to load their Ignition config from a URL. That url is the\
+ `https://<internal-api-vip>/config/master` which is still not available until\
+the **bootstrap** VM is exposing it. It takes few minutes till it does.
+
+With the machine config available the masters pull their Ignition and boots up\
+starting few static pods that will allow:
+1. Internal DNS service for the cluster
+2. `keepalived` to ensure highly available `DNS_VIP` and `API_VIP`
+3. The rest of the openshift cluster pods
+
+For the following subjects please refer to ['baremetal IPI networking infrastructure'][baremetal-ipi-networking]:\
+ - Load-balanced control plane access
+ - API Virtual IP
+ - DNS Resolution During Bootstrapping
+ - DNS Resolution Post-Install
+ - Ingress High Availability
+
+### Risks and Mitigations
+
+- Storage
+
+    Recommended storage setup?
```

Comment:
@mykaul I want to write down a word about recommended storage setup. Can you help? is there anything to take into account?